app/views/staff/shared/activities/_activity.html.haml

```diff
@@ -12,7 +12,7 @@
     %dd.govuk-summary-list__value
       = activity_presenter.identifier
     %dd.govuk-summary-list__actions
-      - if step_is_complete_or_next?(activity: activity_presenter, step: :identifier)
+      - if step_is_complete_or_next?(activity: activity_presenter, step: :identifier) && user_can_edit?(current_user, activity_presenter)
```

Comment:
I left a comment on an earlier commit that added `user_can_edit?` about the potential to move this into Pundit.